server/lib/transforms/html/unfurl-video.js

```diff
@@ -22,7 +22,19 @@ const unfurlVideo = videoId => getVideo(videoId).then(videoTemplate);
 module.exports = ($, options = {}) => {
 	const promises = [];
 	const queueUnfurl = (el, videoId) => videoId && promises.push(
-		unfurlVideo(videoId).then(video => el.replaceWith(video))
+		unfurlVideo(videoId).then(
+			video => el.replaceWith(video),
+			e => {
```

Comment:
previously this error was bubbling up to the article controller and 500ing the whole thing 😱

Reply:
We were doing something very similar on t'app side recently...